apps/site/pages/en/learn/asynchronous-work/understanding-processnexttick.md

```diff
@@ -31,25 +31,27 @@ Use `nextTick()` when you want to make sure that in the next event loop iteratio
 console.log('Hello => number 1');
```

Comment:
To clarify, The entire "Order of events" section should probably be replaced with a link to the event loop documentation.

Like:
```
The execution order of `setImmediate()` and `setTimeout()` can very based on the execution contexts. For more information, refer to [`setImmediate()` vs `setTimeout()`](https://nodejs.org/en/learn/asynchronous-work/event-loop-timers-and-nexttick#setimmediate-vs-settimeout).
```